parachain/Cargo.lock

```diff
@@ -2676,8 +2676,8 @@ dependencies = [
  "sp-keyring",
  "sp-runtime",
  "sp-std",
- "ssz-rs",
- "ssz-rs-derive",
+ "ssz_rs",
```

Comment:
Also, from the looks of the test scripts, we're not actually using this lock file any more (except maybe in language servers). We only build in the `polkadot` & `cumulus` repos now, so the [lockfile in our Cumulus fork](https://github.com/Snowfork/cumulus/blob/1a8a815c224fa51a5df00a92eb28235b42defda6/Cargo.lock) needs this update in a companion PR on that repo.

Reply:
This shouldn't impact testing, as the Cumulus build will still use the updated `Cargo.toml`s here. But it'll change the lockfile in Cumulus, so we should commit it there.